test/insertStyle.test.ts

```diff
@@ -0,0 +1,41 @@
+import test from "ava";
+import insertStyle from "../src/insertStyle";
+import { Browser } from "happy-dom";
+
+const expectA = "body{color:red}";
+
+test("should insertStyle works", async (t) => {
+  const browser = new Browser();
+  const page = browser.newPage();
+
+  page.url = "https://example.com";
+  page.content = `<html><head></head><body></body></html>`;
+
+  // @ts-expect-error
+  global["window"] = page.mainFrame.window;
+  // @ts-expect-error
+  global["document"] = page.mainFrame.window.document;
+
+  const cssStr = insertStyle(expectA);
+
+  const styleSheet = document.head.querySelector("style")!;
+  t.is(
+    styleSheet.textContent,
+    cssStr!,
```

Comment:
Erroneous `!` char.

Reply:
`cssStr` is `string  | undefined` and `t.is` accepts `string | null` so the not null assertion is required to remove `undefined` from `cssStr `  to make it assignable to `t.is` second parameter type

Reply:
If you prefer I can switch to null coalescing:
```suggestion
    cssStr ?? null,
```